<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesPaneNetwork</class>
 <widget class="QWidget" name="PreferencesPaneNetwork">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupbox_proxy">
     <property name="title">
      <string>Proxy Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="checkbox_use_proxy">
        <property name="text">
         <string>Use a &amp;network proxy to connect to the internet</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>HTTP Proxy &amp;URL:</string>
          </property>
          <property name="buddy">
           <cstring>lineedit_proxy_url</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineedit_proxy_url"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolbutton_reset_proxy">
          <property name="toolTip">
           <string>Reset to default value</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qt-resources/qt_resources.qrc">
            <normaloff>:/tango_undo_16.png</normaloff>:/tango_undo_16.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Proxy URLs typically take the form http://example.net:8080/ . GPlates does not support &quot;proxy.pac&quot; proxy autoconfiguration URLs.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Server Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Restart GPlates for the server settings to take effect.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="port_spinbox">
          <property name="minimum">
           <number>1025</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>9777</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="reset_port_button">
          <property name="toolTip">
           <string>Reset to default value</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qt-resources/qt_resources.qrc">
            <normaloff>:/tango_undo_16.png</normaloff>:/tango_undo_16.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="listen_local_checkbox">
        <property name="text">
         <string>Listen on local address only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qt-resources/qt_resources.qrc"/>
 </resources>
 <connections/>
</ui>
